src/VisualStudio/Core/Def/Implementation/Library/FindResults/TreeItems/MetadataDefinitionTreeItem.cs

```diff
@@ -1,70 +0,0 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System.Threading;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Navigation;
-
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.Library.FindResults
-{
-    internal class MetadataDefinitionTreeItem : AbstractTreeItem
```

Comment:
The handling of metadata definitions vs source definitions is handled now at the DefaultDefinitionsAndReferencesFactory layer.